extensions/azurePublishNew/src/node/backgroundProcessManager.ts

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import { ProcessStatus } from '@botframework-composer/types';
+import { v4 as uuid } from 'uuid';
+
+export interface ProcessList {
+  [key: string]: ProcessStatus;
+}
+
+export class BackgroundProcessManager {
```

Comment:
a class with a bunch of statics sounds like a good use for a function

Reply:
yep - I ported this over but haven't rewritten yet.